memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[{"apiVersion":"cache.example.com/v1alpha1","kind":"Memcached","metadata":{"name":"example-memcached"},"spec":{"size":3}}]'
     capabilities: Basic Install
   name: memcached-operator.v0.5.0
-  namespace: placeholder
+  namespace: memcached-operator
```

Comment:
Can we keep this as `default`.
If we're to use this CSV as the example for the getting-started guide then you'll notice we're running everything in the `default` namespace for that walkthrough.